<launch>
  <arg name="arena_info_static_topic" value="/arena_info_static" />
  <arg name="arena_info_dynamic_topic" value="/arena_info_dynamic" />

  <include file="$(find phy_simulator)/launch/joy_ctrl.launch"/>

  <arg name="playground" value = "hard_cases" />
  <!-- <arg name="playground" value = "highway_v1.0" /> -->


  <node pkg="phy_simulator" name="hard_case_phy_simulator_planning_node" type="hard_case_phy_simulator_planning_node" output="screen">
    <param name="vehicle_info_path" type="string" value="$(find playgrounds)/$(arg playground)/vehicle_set.json" />
    <param name="map_path" type="string" value="$(find playgrounds)/$(arg playground)/obstacles_norm.json" />
    <param name="lane_net_path" type="string" value="$(find playgrounds)/$(arg playground)/lane_net_norm.json" />
    
    <!-- <param name="case_path" type="string" value="/home/rancho/2lidelun/Huawei-dataset/DATA/case_old/3964-2022-06-25-161842" /> -->
    <param name="case_path" type="string" value="/home/rancho/2lidelun/Huawei-dataset/DATA/case_old/4812-2022-07-08-172800" />
    
    <param name="case_id" type="int" value="0" />
    <param name="start_frame" type="int" value="0" />
    <param name="init_angle" type="double" value="-100.0" />
    
    <remap from="~arena_info_static" to="$(arg arena_info_static_topic)"/>
    <remap from="~arena_info_dynamic" to="$(arg arena_info_dynamic_topic)"/>
  </node>
</launch>


